addons/website/__manifest__.py

```diff
@@ -237,6 +237,7 @@
             'website/static/src/snippets/s_rating/options.js',
```

Comment:
Commit message:
- `[FIX] website: fixed banner snippet number of columns selection` 
  - commit titles should start in a way that they complete the sentence "When applied, this commit will ..." so here, "fixed" is not right => fix :wink: 
  - put the snippet name with a capital letter and between quotes => "Banner"
  - `number of columns selection` => write "column count option" instead, since it is the name of the option
  
  => In the end, write something like  `[FIX] website: fix the "Banner" snippet column count option` :slightly_smiling_face: 

- Add some steps to reproduce
- The line before the last is too long => max 72 chars
- I would reformulate the commit like this to be more precise :

  > Before this, commit, it was not possible to choose the number of columns of the "Banner" snippet with the "Column Count" option. Indeed, the `s_nb_column_fixed` class that is present on its row prevents this option to be used.
  >   
  >   This commit fixes this issue by removing this class from this snippet row, allowing to use this option.
  >   
  >   Steps to reproduce:
  >   - Drop a "Banner" snippet.
  >   - Change its layout from "Grid" to "Cols"
  >   => the "Column Count" option is not present next to the "Cols" button, preventing to change the number of columns.

  Feel free to reformulate like this, otherwise, change this from yours:
  - `...where the number of column per row selection for banner snippets...` 
    -> `.... where the "Column Count" option in the "Banner" snippet...`
  - `that disabled the column options.`
    -> `that disables this option.`
  - `This class has been removed through javascript in this commit to enable the visibility of column selection.`
    -> `This commit removes this class to make this option visible.`
  - steps to reproduce